src/rpc/cache.cpp

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2021 The Pocketcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <rpc/cache.h>
+#include <rpc/server.h>
+
+static const unsigned int MAX_CACHE_SIZE_MB = 64;
+
+RPCCache::RPCCache() 
+{
+    m_blockHeight = chainActive.Height();
+    m_maxCacheSize = gArgs.GetArg("-rpccachesize", MAX_CACHE_SIZE_MB) * 1024 * 1024;
```

Comment:
Not limited from above?

Reply:
This is a setup - in general, if a person wants to expose an indecent lot, then why not